src/tracker-component.js

```diff
@@ -263,24 +269,15 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
     const currentUrl = new URL(window.location.href);
     const debug = currentUrl.searchParams.get('ppDebug');
     const enableSafari = currentUrl.searchParams.get('ppEnableSafari');
-    
-    if (debug) {
-        // eslint-disable-next-line no-console
-        console.log('PayPal Shopping: debug mode on.');
-    }
+    // eslint-disable-next-line no-console
+    debug && console.log('PayPal Shopping: debug mode on.');
     
     const isSafari = (/^((?!chrome|android).)*safari/i).test(navigator.userAgent);
-
-    if (debug && isSafari) {
-        // eslint-disable-next-line no-console
-        console.log('PayPal Shopping: Safari detected.');
-    }
-
-    if (debug && isSafari && enableSafari) {
-        // eslint-disable-next-line no-console
-        console.log('PayPal Shopping: Safari trackers enabled.');
-    }
```

Comment:
this is correct. you removed the wrong fields when resolving merge conflict.

Reply:
fixed